content/get-involved/index.en.md

```diff
@@ -16,6 +16,7 @@ category: get-involved
 * The official Node.js Twitter account is [nodejs](https://twitter.com/nodejs).
 * The [Node.js Foundation calendar](https://nodejs.org/calendar) with all public team meetings.
 * [Node.js Collection](https://medium.com/the-node-js-collection) is a collection of community-curated content on Medium.
+* The [Community Committee](https://github.com/nodejs/community-committee) is a top-level committee in the Node.js Foundation focused on community-facing efforts.
```

Comment:
I thought CommComm has been retired and it's now all under TSC?

Reply:
You're correct. Let me remove this.